crates/accelerate/src/synthesis/clifford/random_clifford.rs

```diff
@@ -132,10 +132,10 @@ pub fn random_clifford_tableau_inner(num_qubits: usize, seed: Option<u64>) -> Ar
 
     // Apply qubit permutation
     for i in 0..num_qubits {
-        replace_row_inner(table.view_mut(), i, table2.slice(s![i, ..]));
+        replace_row_inner(table.view_mut(), i, table2.slice(s![perm[i], ..]));
```

Comment:
Maybe it's obvious and my brain is still on holiday, but why does this swapping of `perm[i]` and `i` fix the issue?

Reply:
This is because with the _correct_ algorithm we should permute the rows of `table` using the permutation `perm` (see the last page https://arxiv.org/pdf/2003.09412).

Reply:
Ah great, thanks for the link 🙂